### PR TITLE
Update FR locales

### PR DESCRIPTION
**Pull request by thibaudgg**
This patch updates the French locales to be in line with the latest changes in the English ones.

More details about the changes can be found in the v4 upgrade guide: https://github.com/activeadmin/activeadmin/pull/8182

### CODE REVIEW

**Review by javierjulio — COMMENTED** [review]
@thibaudgg thanks! We are just missing the following Kaminari pagination updates here. Would you be ok with translating and adding these in?

```
en:
  views:
    pagination:
      previous: "Previous"
      next: "Next"
```

The reason for the override is to remove the HTML entities that Kaminari adds to just use text. We only need to translate these two words since we don't use the first/last links now.

File: config/locales/fr.yml [hunk 1]
```diff
@@ -26,23 +26,25 @@ fr:
...
       empty: "Aucun(e) %{model} trouvé(e)"
-      one: "Affichage de <b>1</b> %{model}"
-      one_page: "Affichage des <b>%{n}</b> %{model}"
-      multiple: "Affichage de %{model} <b>%{from}&nbsp;-&nbsp;%{to}</b> sur un total de <b>%{total}</b>"
-      multiple_without_total: "Affichage de %{model} <b>%{from}&nbsp;-&nbsp;%{to}</b>"
-      per_page: "Par page: "
+      one: "Affichage de <b>1</b> sur <b>1</b>"
+      one_page: "Affichage de <b>tous les %{n}</b>"
+      multiple: "Affichage de <b>%{from}-%{to}</b> sur <b>%{total}</b>"
+      multiple_without_total: "Affichage de <b>%{from}-%{to}</b>
```
[the remark this answers is not in the document]
Comment (javierjulio) [reply config/locales/fr.yml:46]: @tagliala yes but I believe it is already through the test suite. I encountered it when making locale changes for v4. Ah yes, [this is the test file](https://github.com/activeadmin/activeadmin/blob/master/spec/locales/i18n_spec.rb) that runs the task. Would that be the equivalent or what you were thinking of? I'm not familiar with i18n-tasks so for example, I don't know if we are running it fully in that test or in limited form.

[the remark this answers is not in the document]
Comment (tagliala) [reply config/locales/fr.yml:46]: Ah sorry yes, it is already here

[the remark this answers is not in the document]
Comment (javierjulio) [reply config/locales/fr.yml:46]: No worries, thank you!

[the remark this answers is not in the document]
Comment (thibaudgg) [reply config/locales/fr.yml:46]: @javierjulio whoop, thanks for the syntax fix. 👍🏻

**Comment by thibaudgg** [comment]
@javierjulio just added the `views.pagination` and `activerecord` translations.

File: config/locales/fr.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (javierjulio) [reply config/locales/fr.yml]: Thank you!

**Review by javierjulio — APPROVED** [review]
@thibaudgg thank you ❤️

**Pull request merged**